src/main/java/swinglib/JLabelBuilder.java

```diff
@@ -37,7 +42,9 @@ public JLabel build() {
     Preconditions.checkNotNull(text);
     Preconditions.checkState(!text.trim().isEmpty());
 
-    final JLabel label = new JLabel(text);
+    final String truncated = maxTextLength > 0 && text.length() > maxTextLength ? text.substring(0, 25) + "..." : text;
```

Comment:
Just curious why the second argument to `text.substring()` is `25` and not `maxTextLength` (or something based on `maxTextLength`)?